src/components/DigitalCredentials/dialogs/CredentialNotReceivedDialog.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-dialog
+    id="credential-not-received-dialog"
+    v-model="dialog"
+    width="45rem"
+    persistent
+    :attach="attach"
+    content-class="credential-not-received-dialog"
+  >
+    <v-card class="pa-5">
+      <v-card-text>
+        <p
+          id="dialog-title"
+          class="warning-title"
+        >
+          Didn't receive a credential offer?
+        </p>
+        <p
+          id="dialog-text"
+          class="warning-text"
+        >
+          Credential offers can take up to a minute, depending on your internet connection. Try the following:
+          <ul>
+            <li>
+              <p>Check your intenet connection</p>
+            </li>
+            <li>
+              <p>Try using your mobile data if available</p>
```

Comment:
I've found that Vuetify attaches whitespace to paragraph elements, which I often override. If these list items look OK, just resolve this comment. Otherwise, I'd be tempted to remove the paragraph tags.

Reply:
Resolving this conversation. Instead, see other conversation below re: whitespace between text and list.